Remove active link to Code.Sample project (#404)
microsoft/reverse-proxy: docs/docfx/articles/proxyhttpclientconfig.md

```
Note that instead of defining certificate location as it was in [CertificateConfigData](xref:Microsoft.ReverseProxy.Configuration.Contract.CertificateConfigData) model, this type exposes a fully constructed [X509Certificate](xref:System.Security.Cryptography.X509Certificates.X509Certificate) certificate. Conversion from the configuration contract to the abstraction model is done by a [IProxyConfigProvider](xref:Microsoft.ReverseProxy.Service.IProxyConfigProvider) which loads a client certificate into memory.

> The following is an example of `ProxyHttpClientOptions` usage based on a code-first configuration sample project `ReverseProxy.Code.Sample`. An instance of `ProxyHttpClientOptions` is assigned to the [Cluster.HttpClientOptions](xref:Microsoft.ReverseProxy.Abstractions.Cluster.HttpClientOptions) property before passing the `Cluster` array to `LoadFromMemory` method.

Reviewer: ```suggestion
The following is an example of `ProxyHttpClientOptions` using [code based](xref:configproviders) configuration. An instance of `ProxyHttpClientOptions` is assigned to the [Cluster.HttpClientOptions](xref:Microsoft.ReverseProxy.Abstractions.Cluster.HttpClientOptions) property before passing the `Cluster` array to `LoadFromMemory` method.
```
